content/en/integrations/faq/how-to-import-datadog-resources-into-terraform.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Importing Datadog Resources into Terraform
+kind: faq
+---
+
+## Overview
+
+Terraform supports an out-of-the-box way to import existing resources into your terraform state via the [`terraform import`][1] command.
```

Comment:
Link doesn't seem to render properly.

Reply:
Ha, I think a pre-commit did this, I was using []() link syntax. Will fix